Misc/NEWS.d/next/Core and Builtins/2022-09-16-19-02-40.gh-issue-95778.cJmnst.rst

```diff
@@ -0,0 +1,3 @@
+When :exc:`ValueError` is raised if an integer is larger than the limit,
+mention the :`sys.set_int_max_str_digits` function in the error message.
```

Comment:
the docs build passed, is a bare : shorthand for something in ReST? I'd have written :func: for this.

Reply:
Oops, right, it's a typo: fixed.